client/src/components/DataFiles/DataFilesModals/DataFilesUploadModalListing/DataFilesUploadModalListingTable.jsx

```diff
@@ -5,19 +5,25 @@ import { LoadingSpinner, InlineMessage, Button } from '_common';
 import { FileLengthCell } from '../../DataFilesListing/DataFilesListingCells';
 import { useUpload } from 'hooks/datafiles/mutations';
 import styles from './DataFilesUploadModalListingTable.module.scss';
+import { useSelector } from 'react-redux';
 
 const DataFilesUploadStatus = ({ i, removeCallback, rejectedFiles }) => {
   if (rejectedFiles.filter((f) => f.id === i).length > 0) {
     return <InlineMessage type="error">Exceeds File Size Limit</InlineMessage>;
   }
+  const errorMessage = useSelector((state) => state.files.error.message);
   const status = useUpload().status[i];
   switch (status) {
     case 'UPLOADING':
       return <LoadingSpinner placement="inline" />;
     case 'SUCCESS':
       return <span className="badge badge-success">SUCCESS</span>;
     case 'ERROR':
-      return <InlineMessage type="error">Upload Failed</InlineMessage>;
+      return (
+        <InlineMessage type="error">
+          Upload Failed: {errorMessage}
+        </InlineMessage>
+      );
```

Comment:
Is this mean to be included in a separate PR?

Reply:
Nope. I'll remove it.